lib/src/base-styles/_nx-colors.scss

```diff
@@ -328,6 +345,15 @@
     --nx-color-policy-threat-slider-background-disabled: var(--nx-color-interactive-border-disabled);
     --nx-color-policy-threat-slider-text-disabled: var(--nx-color-policy-threat-slider-border-disabled);
     --nx-color-policy-threat-slider-text: var(--nx-swatch-indigo-97);
+
+    // Radio Checkbox
+    --nx-color-radio-disabled-border: var(--nx-color-disabled);
```

Comment:
> As per Ross' comment above, you can remove this and line 355 since they're values don't change

Thanks! removed here: 34c92ae8e9cd7e3092d4f05218ec3a5b1414a659